### DISCUSSION TITLE
Index.js content showing on the hosted link Please look into it

### DISCUSSION THREAD

**Discussion by Krushnx** [vercel/community#4409] [Help] [answered]
### Index.js content showing on the hosted link Please look into it

Hey i'm trying to upload server part of my mern application on Vercel 
i updated vercel.json couple of times but still only content on index.json is showing in the hosted link

Hosted link : https://mern-auth-sigma.vercel.app/

i've also made the repo public for it 
here is the link https://github.com/Krushnx/mern-auth

@amyegan  @leerob Please look into it

### Index.js content showing on the hosted link Please look into it

Hey i'm trying to upload server part of my mern application on Vercel 
i updated vercel.json couple of times but still only content on index.json is showing in the hosted link

Hosted link : https://mern-auth-sigma.vercel.app/

i've also made the repo public for it 
here is the link https://github.com/Krushnx/mern-auth

@amyegan  @leerob Please look into it

**Comment by amyegan** [comment, marked as answer]
Hi @Krushnx! You need to let the builder know that `index.js` should be a serverless function. You can do that by moving it into a `/api` directory at the root of the `server` project. 

https://vercel.com/docs/functions/serverless-functions/runtimes/node-js
